### PR TITLE
Demo to create user profile document upon a new user account creation…

### PR DESCRIPTION
**Pull request by gireeshp**
… - issue#288

### CODE REVIEW

File: python/create_user_profile/main.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import os
+import json
+
+from appwrite.client import Client
+from appwrite.services.database import Database
+
+user_collection_id = "617497ffb09f7"
```
Comment (Meldiron) [inline python/create_user_profile/main.py:7]: Let's store this in environment variable as it changes for everyone.

File: python/create_user_profile/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+## Introduction
+This function will be triggered at the new user account creation event. This will create the 'users' collection with the newly registered user's profile information.
+
+## Steps
+* Run below code to:
+    * Point PIP target to .appwrite directory in the current working directory
+    * Install appwrite library from requirements.txt
+```
+PIP_TARGET=./.appwrite pip3 install -r ./requirements.txt --upgrade --ignore-installed
+```
```
Comment (Meldiron) [inline python/create_user_profile/README.md:10]: Let's use readme file from some of the existing functions and let's stick to it's structure.

**Review by Meldiron — CHANGES_REQUESTED** [review]
Requested a few changes

File: python/create_user_profile/main.py (hunk 1 above)
Comment (gireeshp) [reply python/create_user_profile/main.py]: Done

File: python/create_user_profile/README.md (hunk 2 above)
Comment (gireeshp) [reply python/create_user_profile/README.md]: Done

**Comment by gireeshp** [comment]
Hi @Meldiron - suggested changes are implemented. Please take a look.

File: python/create_user_profile/main.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+    userId = payload["$id"]
+    userName = payload["name"]
+    email = payload["email"]
+
+
+    client = init_client()
+    database = Database(client)
+
+    database.create_document(user_collection_id, {'user_id': userId, 'user_name': userName, 'email': email}, read=['*'], write=['*'])
```
Comment (Meldiron) [inline python/create_user_profile/main.py:28]: Let's give write permissions only to the user, otherwise hackers would remove them all 😅

**Review by Meldiron — CHANGES_REQUESTED** [review]

File: python/create_user_profile/main.py (hunk 3 above)
Comment (gireeshp) [reply python/create_user_profile/main.py]: Hi @Meldiron  - removed the wildcard write permission.

**Comment by gireeshp** [comment]
Hello @Meldiron  - can you check this pls?

**Review by Meldiron — APPROVED** [review]
Looks good to me

**Pull request merged**